Makefile

```diff
@@ -0,0 +1,41 @@
+
+###########
+# CODEGEN #
+###########
+
+.PHONY: codegen-cli-docs
+codegen-cli-docs: ## Build CLI docs
+	@rm -rf ./src/content/docs/docs/kyverno-cli/reference/kyverno*.md
+	@docker run --user root -v ${PWD}:/work --rm ghcr.io/kyverno/kyverno-cli docs	\
+		--autogenTag=false															\
+		--website																	\
+		--noDate																	\
+		--markdownLinks																\
+		--output "/work/content/docs/docs/kyverno-cli/reference"
+
+.PHONY: codegen-policies
+codegen-policies: ## Render policies
+	@rm -rf ./src/content/docs/policies/*/
+	@cd render && go run . -- https://github.com/kyverno/policies/main ../src/content/docs/policies/
+
+.PHONY: codegen
+codegen: ## Rebuild all generated code and docs
```

Comment:
@lucchmielowski I was thinking maybe there is still a need to generate cli and policies docs and run checks in CI. Any thoughts?